### PR TITLE
Reconcile branches

### PR DESCRIPTION
**Pull request by Rotorsoft**
## Link to Issue
Closes: #7806

## Description of Changes
- Reconciles typing fixes in pnpm branch

### CODE REVIEW

**Review by kurtisassad — APPROVED** [review]

File: packages/commonwealth/server/routes/logout.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ const logout = async (models: DB, req: Request, res: Response) => {
   // early, so we also call req.session.destroy() and clear the
   // session cookie before returning
   stats().decrement('cw.users.logged_in');
-  req.logout();
+  req.logout(() => {});
```
Comment (timolegros) [inline packages/commonwealth/server/routes/logout.ts:10]: This makes me a little worried. Does this call even do anything without a callback? Does sign-out behaviour change if we provide an empty callback?

Is there a reason for this change?

We may want to move the below `req.session.destory` logic into the `req.logout` callback per: https://www.passportjs.org/concepts/authentication/logout/.

Comment (Rotorsoft) [reply packages/commonwealth/server/routes/logout.ts:10]: good point... this is pnpm complaining about not having the explicit callback

Comment (Rotorsoft) [reply packages/commonwealth/server/routes/logout.ts:10]: @timolegros tested your proposal but it doesn't work, seems like the comment is there for a reason. Having the empty callback doesn't break logout either, and makes pnpm happy

**Review by timolegros — APPROVED** [review]

File: packages/commonwealth/server/routes/logout.ts (hunk 1 above)
Comment (Rotorsoft) [reply packages/commonwealth/server/routes/logout.ts]: just upgraded all passport deps to the same version and removed the callback in this PR... will deal with this issue again in final PR

**Pull request merged**